addon/components/sub-form.hbs

```diff
@@ -34,7 +55,7 @@
       </AuToolbar>
     {{/if}}
 
-    <div class="au-o-flow">
+    <div class="au-o-flow {{if this.collapsed 'au-u-hidden'}}">
```

Comment:
using au-u-hidden here because it seems that rendering it conditionally breaks the children of a propertygroup